recipes/precision-recall-gain/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "precision-recall-gain" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/precision-recall-gain-{{ version }}.tar.gz
+  sha256: 3431e02edcbe7b2254b53ab4447e42e409dea8060efaf259e8378eb9e23816d1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python 3.11
+    - setuptools >=30.3.0
+    - wheel
+    - setuptools-scm >=3.3.1
+    - cython
+    - pip
+  run:
+    - python >=3.9
```

It can't be lower than the one you are using to build!

```suggestion
    - python >={{ python_min }}
```